ui/app/templates/components/mfa/mfa-login-enforcement-form.hbs

```diff
@@ -10,7 +10,7 @@
     spellcheck="false"
     value={{@model.name}}
     disabled={{not @model.isNew}}
-    class="input field"
+    class="input field {{if this.errors.name.errors 'has-error-border'}}"
```

Comment:
this adds the red border around the name field on error. Noticed it was missing here and went ahead and added it while I was in there.